src/modules/blockchainApplication/manage/store/actionTypes.js

```diff
@@ -3,8 +3,6 @@ const actionTypes = {
   addApplicationByChainId: 'ADD_APPLICATION_BY_CHAIN_ID',
   deleteApplicationByChainId: 'DELETE_APPLICATION_BY_CHAIN_ID',
   setCurrentApplication: 'SET_CURRENT_APPLICATION',
-  setApplicationNode: 'SET_APPLICATION_NODE',
-  setApplications: 'SET_APPLICATIONS',
```

Comment:
@eniolam1000752 This (`setApplications`) seems to have been accidentally removed and should be reverted